news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "All you need to know about built-in modules"
+subtitle: "Get updated with the recent work done to highlight PrestaShop modules"
+date: 2020-01-18 12:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+
+---
+
+Do you know that PrestaShop 1.7.7 comes with 88 built-in modules? That is 88 ways for merchants to start their online store and empower their business, directly from their back office! :fire:
+
+In this article, we will try to answer all your questions about this topic. So let’s get started!
+
+## What is a built-in module exactly?
+
+A built-in module is a module that was developed by PrestaShop and that is available in your back office either in the Module Manager or in the Module Catalog. 
+
+We are not talking about "native modules" here, as the definition of a native module is a bit more complicated. Indeed, some native modules are present in the installation zip, some are pre-installed and some others are not. So, to make it easier, we have chosen to talk about built-in modules in this article.
+
+To know more about native modules, you can check the [documentation](https://devdocs.prestashop.com/1.7/development/native-modules/). 
```

Comment:
```suggestion
To know more about native modules, you can check the [documentation](https://devdocs.prestashop.com/1.7/development/native-modules/).

### New native modules distribution

Following an opinion poll carried out through the [User Club](https://www.prestashop.com/en/club) and the usage studied along with the project's maintainers, we made the decision to redefine the current native modules distribution. Some have not been updated for years, some did not work with PrestaShop 1.7, and others could be merged into a single module, this is why a small number of modules have just been archived. It means that they are still available on GitHub (although they have been removed from the organization) but the Core team no longer maintains them.
```

Reply:
@marionf, @eternoendless, feel free to modify this part if need be, thanks!

Reply:
I just added that they can be forked :)